index.html

```diff
@@ -4,8 +4,90 @@
+            <div class="nav-header">
+                <div class="nav-title">Layout 3</div>
+            </div>
+            <div class="nav-btn">
+                <label for="nav-check">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </label>
+            </div>
+
+            <div class="nav-links">
+                <a href="#">about</a>
+                <a href="#">Service</a>
+                <a href="#">contact</a>
+            </div>
+        </div>
+
+
+
+        <div class="child">
+            <div class="container1">
+                <h1>Sample Heading</h1>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Necessitatibus debitis, ipsam nobis nihil
+                    mollitia voluptas voluptate eligendi possimus doloremque at ex sed eveniet laudantium placeat
+                    pariatur sapiente sequi quae harum autem cumque, laborum est assumenda dolor?
+                </p>
+                <div class="bluebtn">Call to action!</div>
+            </div>
+            <hr>
+            <div class="subheading">
+                <h2>Latest Features</h2>
+            </div>
+
+            <div class="container2">
+                <div class="gallery box1">
+                    <img src="./31343C.svg" alt="img">
+                    <div class="gallerytext">
+                    <h2>Feature Lable</h2>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    </div>
+                    <div class="bluebtn">Buy Now!</div>
+                    <div class="whitebtn">More Info</div>
+                </div>
+                <div class="gallery box2">
+                    <img src="./31343C.svg" alt="img">
+                    <div class="gallerytext">
+                    <h2>Feature Lable</h2>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    </div>
+                    <div class="bluebtn">Buy Now!</div>
+                    <div class="whitebtn">More Info</div>
+                </div>
+                <div class="gallery box3">
+                    <img src="./31343C.svg" alt="img">
+                    <div class="gallerytext">
+                    <h2>Feature Lable</h2>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    </div>
+                    <div class="bluebtn">Buy Now!</div>
+                    <div class="whitebtn">More Info</div>
+                </div>
+                <div class="gallery box4">
+                    <img src="./31343C.svg" alt="img">
+                    <div class="gallerytext">
+                    <h2>Feature Lable</h2>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</p>
+                    </div>
+                    <div class="bluebtn">Buy Now!</div>
+                    <div class="whitebtn">More Info</div>
```

Comment:
Use link in src instead downloading the image.
rename the image with meaningful name if you have downloaded .
use - as a separator while naming the two word class name like gallery-text